classes/quick-editor/class-quick-editor.php

```diff
@@ -50,119 +68,215 @@ public function end_capture_page() {
-	public function get_avatar_url( $url ) {
-		return add_query_arg( 't', time(), $url );
+	private function get_verified_services_section() {
+		$verified_services = esc_html( __( 'Verified Services', 'gravatar-enhanced' ) );
+		$description = sprintf(
+			/* translators: URL */
+			__( 'Let people know where they can find you online. <a href="%s">Edit your verified accounts.</a>', 'gravatar-enhanced' ),
+			'https://gravatar.com/profile/verified-accounts'
+		);
+
+		return <<<HTML
+<tr class="user-verified-services">
+	<th>$verified_services</th>
+	<td>
+		<div id="gravatar-verified-services" class="gravatar-profile__loading"></div>
+		<p class="description">$description</p>
+	</td>
+</tr>
+HTML;
 	}
 
 	/**
-	 * Replace the text link with a button
-	 *
 	 * @return string
 	 */
-	public function add_quick_editor_link() {
+	private function get_gravatar_card_section() {
 		$current_user = $this->get_user();
-
 		if ( ! $current_user ) {
 			return '';
 		}
 
-		$profile_details = esc_html( $current_user->user_email );
-		$what_is_gravatar = esc_html__( 'What is Gravatar?', 'gravatar-enhanced' );
-		$mail_icon = plugins_url( 'resources/mail.svg', GRAVATAR_ENHANCED_PLUGIN_FILE );
-		$gravatar_icon = plugins_url( 'resources/gravatar.svg', GRAVATAR_ENHANCED_PLUGIN_FILE );
+		$title = esc_html( __( 'Gravatar Card', 'gravatar-enhanced' ) );
+		$description = __( 'Your Gravatar card is shown in comments and profile blocks. Gravatar profiles travel with you across millions of sites, showing your photo, bio, and links - your digital business card.', 'gravatar-enhanced' );
+		$edit = esc_html( __( 'Edit your Gravatar card', 'gravatar-enhanced' ) );
 		$avatar_url = get_avatar_url( $current_user->ID );
 
-		$html = <<<HTML
-</p>
-<div class="gravatar-account">
-	<div>
-		<img src="$mail_icon" alt="" />
-		$profile_details
-	</div>
-	<div>
-		<a target="_blank" href="https://gravatar.com">$what_is_gravatar</a>
-
-		<img src="$gravatar_icon" alt="" />
-	</div>
-</div>
-
-<div class="gravatar-hovercard-container">
-	<div class="gravatar-hovercard gravatar-profile__loading">
-		<div class="gravatar-hovercard">
-			<div class="gravatar-hovercard__inner">
-				<div class="gravatar-hovercard__header">
-					<img class="gravatar-hovercard__avatar" src="$avatar_url" width="104" height="104" />
-					<h4 class="gravatar-hovercard__name"></h4>
-				</div>
-				<div class="gravatar-hovercard__body">
-					<p class="gravatar-hovercard__description"></p>
-				</div>
-				<div class="gravatar-hovercard__footer">
-					<div class="gravatar-hovercard__social-links">
+		return <<<HTML
+<tr class="user-gravatar-card">
+	<th>$title</th>
+	<td>
+		<div class="gravatar-hovercard-container">
+			<div class="gravatar-hovercard gravatar-profile__loading">
+				<div class="gravatar-hovercard">
+					<div class="gravatar-hovercard__inner">
+						<div class="gravatar-hovercard__header">
+							<img class="gravatar-hovercard__avatar" src="$avatar_url" width="104" height="104" />
```

Comment:
UX: Lack the `alt` attr.

```suggestion
							<img class="gravatar-hovercard__avatar" src="$avatar_url" width="104" height="104" alt="..." />
```